src/__test__/fixture/index.ts

```diff
@@ -36,7 +36,19 @@ export const fixtureStart = async (
   app.post('/login', (_req, res) => {
     const profile = {
       email: 'john@doe.com',
-      id: 123
+      id: 123,
+      checkField: true
```

Comment:
Could you explain a little bit more of why you're adding this `checkField` property ?

Reply:
The `checkField` property is simulating a claim in the token that the custom method written in `onAuthentication` would check. For eg. tokens might have a `roles` claim and the callback created in `onAuthentication` can check this claim and allow or reject it based on that. The test is checking if the value of `checkField` is true or false.